test/function/samples/empty-string-as-module-name/_config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	description: 'allows using empty string as a valid module name',
+	options: {
```

Comment:
Via `options`, you can basically override all of those defaults the same way you would write a config file (i.e. you can specify output options via `output: { /* some options */}`) . In our case, it is just the external option and the plugins.